pricing-page-exercise/tugba/index.html

```diff
@@ -4,10 +4,114 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="./styles.css">
+<link rel="stylesheet" href="./styles.css">
   <title>Pricing page Exercise</title>
 </head>
 <body>
-  
+ <header>
+    <nav class="navbar">
+        <div><h3 class="heading-small">Company name</h3></div>
```

Comment:
Usually, there's a logo or a company name in the navigation bar on every website. When you click on it, it always takes you to the home page. 
So in this case it would make more sense to use `<a>` element instead of `<h3>`.